craco.config.js

```diff
@@ -16,6 +16,10 @@ module.exports = {
         include: /node_modules/,
         type: "javascript/auto",
       })
+      webpackConfig.resolve.fallback = {
```

Comment:
can we remove this after shifting to window.crypto?

Reply:
Yup, removed in https://github.com/isomerpages/isomercms-frontend/pull/1316/commits/6307c6951eede47ad3e35791c308cf44246db347